README.md

```diff
@@ -1,29 +1,29 @@
 # Unified Catalog Migration
 
-Tool for migrating your Unified Catalog items and assets to Marketplace products.
+A tool for migrating your Unified Catalog items and assets to Marketplace products.
 
 The script will:
 
 1. Log into the platform using the `axway auth login` either using the browser or using a service account (if set in the configuration file)
 2. Read all existing catalog items from a specific environment
 3. For each catalog item
     * Find the linked API Service
-    * Create an asset (or use an existing one) and linked it to the API Service
-    * Create a product (if not existing yet) and linked it to the asset created
-    * (Optional) Publish the product to the desire Marketplace
-    * (Optional if product is published) Read existing Active subscription and create the corresponding Marketplace subscription / application if exists
```

Comment:
* Create a corresponding  sunscription and application in the Marketplace for each active catalog item subscription